index.html

```diff
@@ -1,37 +1,495 @@
+
+                    <li class="footer__topics__item"><a href="/category/e-commerce">E-Commerce</a></li>
+                    <li class="footer__topics__item"><a href="/category/freebies">Freebies</a></li>
+                    <li class="footer__topics__item"><a href="/category/graphics">Graphics</a></li>
+                    <li class="footer__topics__item"><a href="/category/html">HTML</a></li>
+                </ul>
+                <ul class="footer__topics__list1 flex-container flex-row  align-items bg-color">
+                    <li class="footer__topics__item"><a href="/category/illustrator">Illustrator</a></li>
+                    <li class="footer__topics__item"><a href="/category/inspiration">Inspiration</a></li>
+                    <li class="footer__topics__item"><a href="/category/ios">iOS</a></li>
+                    <li class="footer__topics__item"><a href="/category/javascript">JavaScript</a></li>
+                    <li class="footer__topics__item"><a href="/category/mobile">Mobile</a></li>
+                    <li class="footer__topics__item"><a href="/category/pattern-libraries">Pattern Libraries</a></li>
+                    <li class="footer__topics__item"><a href="/category/performance">Performance</a></li>
+                    <li class="footer__topics__item"><a href="/category/photoshop">Photoshop</a></li>
+                    <li class="footer__topics__item"><a href="/category/plugins">Plugins</a></li>
+                    <li class="footer__topics__item"><a href="/category/react">React</a></li>
+                    <li class="footer__topics__item"><a href="/category/responsive-web-design">Responsive Web Design</a></li>
+                </ul>
+                <ul class="footer__topics__list1 flex-container flex-row  align-items">
+                    <li class="footer__topics__item"><a href="/category/service-workers">Service Workers</a></li>
+                    <li class="footer__topics__item"><a href="/category/sketch">Sketch</a></li>
+                    <li class="footer__topics__item"><a href="/category/typography">Typography</a></li>
+                    <li class="footer__topics__item"><a href="/category/ui">UI</a></li>
+                    <li class="footer__topics__item"><a href="/category/usability">Usability</a></li>
+                    <li class="footer__topics__item"><a href="/category/user-experience">User Experience</a></li>
+                    <li class="footer__topics__item"><a href="/category/wallpapers">Wallpapers</a></li>
+                    <li class="footer__topics__item"><a href="/category/web-design">Web Design</a></li>
+                    <li class="footer__topics__item"><a href="/category/wordpress">WordPress</a></li>
+                    <li class="footer__topics__item"><a href="/category/workflow">Workflow</a></li>
+                </ul>
+            </div>
+        </div>
+
+
+
+        <div class="footer__notes bg-color">
```

Comment:
Please add placeholders for the two images on either side of this element. Add placeholders (indicating that there should be an image), not the actual image.

Right now 👇

![Annotation 2020-06-14 093637](https://user-images.githubusercontent.com/57726348/84584514-063d7200-ae23-11ea-96b0-267781f4694e.jpg)

Original 👇

![Annotation 2020-06-14 003715](https://user-images.githubusercontent.com/57726348/84584520-105f7080-ae23-11ea-8d6f-37df1c79aa1c.jpg)